xml/System/Span`1.xml

```diff
@@ -858,7 +870,7 @@ For a `Span<Char>`, the `ToString` method returns a <xref:System.String> that co
         <remarks>
           <format type="text/markdown"><![CDATA[  
 
-If the source and `destination` overlap, this method places the original values in a temporary location before `destination` is overwritten.
+If the source and `destination` overlap, the source is written to `destination` without any loss of data.
```

Comment:
I am not sure if `without any loss of data` is clear. The contents of the destination to get overwritten by whatever was in the source and hence are lost. Thoughts?

Reply:
How about "without any loss of the source data"?